Xamarin.Forms.Platform.Android/Renderers/ShellToolbarTracker.cs

```diff
@@ -272,7 +273,7 @@ protected virtual void OnPagePropertyChanged(object sender, PropertyChangedEvent
 
 		void OnBackButtonBehaviorChanged(object sender, PropertyChangedEventArgs e)
 		{
-			if (!e.Is(BackButtonBehavior.CommandParameterProperty))
+			if(!e.Is(BackButtonBehavior.CommandParameterProperty))
```

Comment:
Can we drop all these extra whitespace changes to make this easier to review?

Reply:
Sure, I have reverted these changes.